management/global/organizations/aws-tf-organization.md

```diff
@@ -0,0 +1,99 @@
+## AWS Terraform Management
+
+### Update AWS Organization features
+1. Go to `management/global/organizations`.
+2. Edit `organization.tf` to enable/disable Organization features
+3. Or edit the `policies_scp.tf` file to add/remove/update Service Control Policies.
+4. Finally, run the Terraform workflow to apply the actual changes.
+
+### Add/remove AWS accounts
+1. Go to `management/global/organizations`.
+2. Edit `locals.tf` to add/remove accounts from the local `accounts` variable.
+3. Finally, run the [Terraform workflow](https://leverage.binbash.com.ar/user-guide/ref-architecture-aws/workflow/) to apply the actual changes.
+4. Add the new account to `config/common.tfvars`.
+5. From here, you may very likely want to create the initial directory structure for this new account *as explained right below*.
+
+### Add/remove/update AWS Organization Service Control Policies
+1. Go to `management/global/organizations`.
+2. Edit the `policies_scp.tf` file to add/remove/update Service Control Policies.
+3. Finally, run the [Terraform workflow](https://leverage.binbash.com.ar/user-guide/ref-architecture-aws/workflow/) to initialize and apply this layer.
+
+### Add/remove AWS IAM user accounts
+For instance, to remove a user from the security account you would need to make the following steps:
+1. Go to `security/global/base-identities`
+2. Remove the user from the users list in `locals.tf` file.
+3. Also remove it from the corresponding groups in `groups.tf` file.
+4. Remove output references from `outputs.tf` file.
+5. Delete the corresponding GPG key from the `keys` subdirectory.
+6. Finally, run the [Terraform workflow](https://leverage.binbash.com.ar/user-guide/ref-architecture-aws/workflow/) to initialize and apply this layer.
+
+And to remove a user from the security account you can follow the same steps but should instead go to this directory: `management/global/base-identities`
+
+### Create the initial directory structure for a new account
+As an example, we will set up the `apps-prd` account by using the `apps-devstg` as a source reference code:
+1. Ensure you are at the root of this repository
+2. Create the initial directory structure for the new account:
+    ```
+    mkdir -p apps-prd/global
+    mkdir -p apps-prd/us-east-1
```

Comment:
this can be summed up as follows:
```
mkdir -p apps-prd/{global,us-east-1}
```
(to have just one command)

Reply:
@juanmatias updated, thanks!